src/components/screens/PointOfInterest.js

```diff
@@ -83,7 +88,7 @@ export const PointOfInterest = ({ data, navigation }) => {
     <View>
       {!!images && images.length > 1 && <ImagesCarousel data={images} />}
 
-      <WrapperWithOrientation orientation={orientation} dimensions={dimensions}>
+      <WrapperWithOrientation>
```

Comment:
now the main components rely on the `OrientationContext` and the `WrapperWithOrientation` as well. can this lead to more re-renders than before, as both react to orientation changes?

Reply:
that should not be the case, as demonstrated here: https://snack.expo.io/@jk.tpwd/rerender-context